### ISSUE TITLE
MySQL: can't parse db.table in GRANT statement

### ISSUE THREAD

**Issue by coanor** [antlr/grammars-v4#1229]
Under GRANT statement

    GRANT SELECT ON db1.table1 to 'me'@'localhost'

the parse failed, it seems that the `privilegeLevel` rule should accept `fullID` syntax like `SELECT  FROM db.table`.

I have tried 

    GRANT SELECT ON `db1`.`table1` to 'me'@'localhost'

and

    GRANT SELECT ON db1.`table1` to 'me'@'localhost'

the parser works well, but 

    GRANT SELECT ON `db1`.table1 to 'me'@'localhost'

still failed too, It seems the table part must be reverse-quoted.

**Comment by mike-lischke** [comment]
The MySQL grammar from Oracle is complete and supports this syntax (including all the changes introduced with server 8.0). If you have the option to switch I recommend to try this out instead. It requires some work on the infrastructure side, if you don't use C++, however.

A few more details about this grammar can be found in the [MySQL folder in this grammar repositiory](https://github.com/antlr/grammars-v4/tree/master/mysql/Oracle).

**Comment by x5engine** [comment]
Yes, indeed! Oracle supports this syntax and all new changes following it... please check the documentation.

**Comment by mike-lischke** [comment]
This is **not** about Oracle, but MySQL! Please keep that in mind.

**Comment by x5engine** [comment]
I understand this is about NoSQL not about Oracle :)
